docs/modules/tiered-storage/pages/capacity-planning.adoc

```diff
@@ -0,0 +1,73 @@
+= Capacity Planning
+
+You must calculate the capacity of external storage you require to ensure that your selected storage is suitable for your needs.
+
+The considerations for memory and disk capacity are explored below.
+
+== Memory Capacity
+
+A HybridLog data structure exists for each partition in all your maps, and for every index defined on the maps.
+As the HybridLog includes a mandatory memory component, the configuration of IMaps backed by Tiered Storage dictates the minimum required native memory for the system.
+This minimum memory footprint can be calculated using the following formula:
+
+----
+min_memory = (ts_imap_count + ts_index_count) * partition_count
+             * min_hybridlog_mem
+----
+
+The variables are as follows:
+
+- `ts_imap_count`: The number of IMaps where Tiered Storage is enabled.
+- `ts_index_count`: The sum of all indices defined for the IMaps where Tiered Storage is enabled.
+- `partition_count`: The partition count of the Hazelcast cluster; by default 271.
+- `min_hybridlog_mem`: Four times the size of the hybrid log page (by default 4*1MB=4MB).
+
+NOTE: There is a HybridLog instance for every configured index on every partition.
+
+This means that if you have Tiered Storage enabled for a single IMap, with no index configured, and the default partition count and page size, the minimum required memory is `(1 + 0) * 271 * 4MB = 1084MB`.
+
+The Hazelcast cluster members validate, on startup, if the native memory configured for the member is greater than or equal to the minimum required memory as calculated above.
+If not, the cluster fails to start.
+
+Moreover, around 20 MB of RAM is required for every one million entries stored on a member, including backups. To calculate it, divide the total expected number of entries (with the backups) stored in the cluster by the member count.
+For example, if you have a cluster of ten members and you need to store five million entries in the cluster, then, with the backup count of 1, you need around 20 MB of RAM on each member.
+
+NOTE: Since the partitions are distributed in the cluster, a single member is not expected to store data from all partitions.
+However, in unexpected circumstances, such as network splits and member shutdowns, all data might be stored in a single member.
+For this reason, Hazelcast always uses the partition count of the cluster for the validation.
+
+TIP: For optimal performance and resource allocation, it is advised to allocate approximately 12GB of memory per 100GB of Tiered Storage.
```

Comment:
This statement is harmless, so no strong objections against having it in the manual, but there is nothing backing this number. I've seen good results with 4GB and with bigger data set, and results where 4GB -> 16GB made only insignificant improvement if anything. What's considered efficient depends on the access pattern (if there is a hot set, and its size) and not on the memory:disk ratio.

Reply:
@avtarraikmo , any comments on this one, should we come up with a flexible statement?

Reply:
We had a chat in the weekly doc meeting with Avtar, he is suggesting to keep this recommendation as it is.